app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
Ok, then I'll set the version to 0.2.0 right away so you don't have to change it. We can also use `-dev` suffix between the releases for the case when between the realeases there are several commits touching the schema file. This way we won't have two different files in our commit history with the same non-dev version and will help to avoid issues when someone uses unreleased source code.

So I commit 0.2.0-dev now, you change it to 0.2.0 when you make a release. If in between 0.2.2 and 0.2.3 there are changes to the schema we change the version to 0.2.3-dev and then change it to 0.2.3 on release of 0.2.3. This way we avoid assigning normal version numbers (0.2.2 and 0.2.3) to intermidiate versions which may differ from both 0.2.2 and 0.2.3 releases.

> maybe we can add a list with schema versions and schema changelog somewhere?

I think we can create a page at https://diaspora.github.io/diaspora_federation.